### PR TITLE
CI: ClickHouse

### PR DESCRIPTION
**Pull request by serprex**

### CODE REVIEW

File: flow/e2e/clickhouse/clickhouse.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+func (s ClickHouseSuite) Suffix() string {
+	return s.suffix
+}
+
+func (s ClickHouseSuite) Peer() *protos.Peer {
+	ret := &protos.Peer{
+		Name: e2e.AddSuffix(s, "click"),
+		Type: protos.DBType_CLICKHOUSE,
+		Config: &protos.Peer_ClickhouseConfig{
+			ClickhouseConfig: &protos.ClickhouseConfig{
```
Comment (serprex) [inline flow/e2e/clickhouse/clickhouse.go:50]: this needs more

File: flow/e2e/clickhouse/clickhouse.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+func (s ClickHouseSuite) DestinationTable(table string) string {
+	return table
+}
+
+func (s ClickHouseSuite) Teardown() {
+	e2e.TearDownPostgres(s)
+}
+
+func (s ClickHouseSuite) GetRows(tableName string, colsString string) (*model.QRecordBatch, error) {
+	// TODO
```
Comment (serprex) [inline flow/e2e/clickhouse/clickhouse.go:69]: TODO

File: flow/workflows/sync_flow.go [hunk 3]
```diff
@@ -85,6 +85,11 @@ func SyncFlowWorkflow(
 		})
 	}
 
+	syncFlowCtx := workflow.WithActivityOptions(syncSessionCtx, workflow.ActivityOptions{
```
Comment (Amogh-Bharadwaj) [inline flow/workflows/sync_flow.go:88]: what's the rationale for moving this up?

File: flow/e2e/clickhouse/clickhouse.go [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+		batch.Records = append(batch.Records, qrow)
+	}
+
+	return batch, rows.Err()
+}
+
+func SetupSuite(t *testing.T) ClickHouseSuite {
+	t.Helper()
+
+	suffix := "ch_" + strings.ToLower(shared.RandomString(8))
```
Comment (Amogh-Bharadwaj) [inline flow/e2e/clickhouse/clickhouse.go:154]: do we want to add timestamp here for future cleanup jobs?

**Review by Amogh-Bharadwaj — APPROVED** [review]

File: flow/workflows/sync_flow.go (hunk 3 above)
Comment (serprex) [reply flow/workflows/sync_flow.go:88]: less allocations, unrelated

File: flow/e2e/clickhouse/clickhouse.go (hunk 4 above)
Comment (serprex) [reply flow/e2e/clickhouse/clickhouse.go:154]: database running in action, no cleanup necessary

**Pull request merged**